nuklear.h

```diff
@@ -4475,10 +4454,8 @@ NK_API struct nk_rect
 nk_recti(int x, int y, int w, int h)
 {
     struct nk_rect r;
-    r.x = (float)x;
-    r.y = (float)y;
-    r.w = (float)w;
-    r.h = (float)h;
+    r.x = (float)x; r.y = (float)y;
+    r.w = (float)w; r.h = (float)h;
```

Comment:
This is IMHO not necessary, but if you like it.

Reply:
This is to make it consistent with `nk_rect`, the function right above it.

Reply:
Yeah I don't really mind this change. It is more cosmetic than anything. Consistency is nice but on its on would not warrant a new version.